src/Model/Filter/Boolean.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Search\Model\Filter;
+
+class Boolean extends Base
+{
+
+    /**
+     * Default configuration.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'truthy' => [1, true, '1', 'true',  'yes', 'on'],
```

Comment:
I disagree :)
It kinda does - https://github.com/cakephp/cakephp/blob/master/src/Filesystem/Folder.php#L37 as only of of many - but it sure does way too less in some cases.
But you can also keep this way.